app/js/components/orderEntry/Orders.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Accordion from './Accordion';
+import allOrdersMock from './allOrdersMock.json';
+import OrderHeader from './OrderHeader';
+import LabOrderDetails from './LabOrderDetails';
+import DrugOrderDetails from './DrugOrderDetails';
+
+const Orders = () => (
+  <React.Fragment>
+    {allOrdersMock.orders.map(order =>
+      (order.type === 'drugorder' ? (
+        <Accordion
+          title={<OrderHeader status={order.status} orderable={order.orderable} />}
+          key={order.id}>
+          <DrugOrderDetails
+            dosingInstructions={order.dosingInstructions}
+            dispense={order.dispense}
+            activeDates={order.activeDates}
+            orderer={order.orderer}
+          />
+        </Accordion>
+      ) : (
+        <Accordion title={<OrderHeader status={order.status} orderable={order.orderable} />} key>
+          <LabOrderDetails urgency={order.urgency} orderer={order.orderer} />
+        </Accordion>
+      )))}
+  </React.Fragment>
```

Comment:
Composing the components above like this can save some lines of codes:
```
<Accordion 
   title={<OrderHeader status={order.status} orderable={order.orderable} />}
   key={order.id}>
   {order.type === 'drugorder' ? 
       <DrugOrderDetails
            dosingInstructions={order.dosingInstructions}
            dispense={order.dispense}
            activeDates={order.activeDates}
            orderer={order.orderer}
       /> :
      <LabOrderDetails urgency={order.urgency} orderer={order.orderer} />}
</Accordion>
```

Reply:
ohh, I'll refactor this..

Reply:
done